pyproject.toml

```diff
@@ -21,13 +21,15 @@ target-version = "py38"
 [tool.ruff.lint]
 extend-select = [
 	"I", # isort
+	"B006", # no mutable default arguments
```

Comment:
there was one violation of this rule, and it can cause actual bugs, so I decided to introduce it